### ISSUE TITLE
Move all tests from unittest to pytest format

### ISSUE THREAD

**Issue by extsoft** [qaivanpleten/sp_learning#2]
```
# class SetUpClass(unittest.TestCase):
#     @classmethod
#     def setUpClass(cls):
#         raise Exception()
#
#     @classmethod
#     def tearDownClass(cls):
#         pass
#
#     def test_bla(self):
#         pass
#
#     def test_unit(self):
#         raise Exception()



@pytest.fixture
def driver():
    print("starting")
    yield
    print('end')


def test_fixt(driver):
    print('test')
    # raise Exception()


@pytest.fixture
def driver2():
    print("starting")
    yield "yield text"
    print('end')


def test_fixt2(driver2):
    print(driver2 + " from test")

```

`py.test -s -v tests`

**Comment by qaivanpleten** [comment]
@extsoft 
I got this error while I'm trying to run test_case using the fixture. 
![image](https://user-images.githubusercontent.com/22939073/30635047-dbc91cf4-9df9-11e7-97d6-69ac8d0f4d73.png)

But, when I imported the fixture, the test is running without errors. 
![image](https://user-images.githubusercontent.com/22939073/30635127-25497806-9dfa-11e7-9687-d1bfee1ddb45.png)

Interesting, that import string looks like Pycharm doesn't use it.
